Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/index.md

```diff
@@ -80,27 +79,27 @@ Profile consists of the following information:
   :::
 
 - Telephone: The contact number of the user.
-- Edit profile: Allows to update and ensure that your information is valid and up to date for your Umbraco Cloud profile.
+- Edit profile: Allows you to update and ensure that your information is valid and up to date for your Umbraco Cloud profile.
   ![Edit profile](images/change-profile-info.png)
 
 - Change Password: Change the password for your Umbraco Cloud account from here.
   ![change password](images/change-password.png)
 
 ## Release Notes
 
-On Umbraco Cloud portal, you can now find the link for the **Release Notes** in the **Profile Settings** menu. Release notes will be published multiple times every month and list the most relevant fixes and features added to the portal.
+On the Umbraco Cloud portal, you can now find the link to the **Release Notes** in the **Profile** dropdown. Release notes are published multiple times every month and list the most relevant fixes and features added to the portal.
```

Comment:
```suggestion
On the Umbraco Cloud portal, you can now find the link to the **Release Notes** in the **Profile** dropdown. Release notes are published every month and list the most relevant fixes and features added to the portal.
```

Reply:
@eshanrnh I think we should just mention that it gets published every month instead of saying multiple times, just so people don't expect that they do it several times a month :)

Reply:
Good point.